multicluster/multi_cluster_cache.go

```diff
@@ -492,7 +492,7 @@ func (mcc *multiClusterCache) convertClusters(clusters []string) ([]string, bool
 }
 
 type multiClusterInformer struct {
-	multi             bool
+	multi             bool // whether this informer is for managed clusters, or only for fed cluster
```

Comment:
change the name to `enableMultiple` or else?

Reply:
fixed